src/components/FileRepo.js

```diff
@@ -17,6 +25,32 @@ const FileRepo = ({ state, effects }) => {
         )}
       </pre>
       {!state.loggedInUser && <div>Seeing files as annon</div>}
+      <Arranger
+        index="file"
```

Comment:
Will this eventually be read in from env?

Reply:
It can be, but the site will have multiple indexes, so if it is an environment variables it would probably be something like `REACT_APP_FILE_INDEX`.